test/Graphics/Transform.cpp

```diff
@@ -28,31 +29,27 @@ TEST_CASE("sf::Transform class - [graphics]")
 
         SUBCASE("3x3 matrix constructor")
         {
-            const sf::Transform transform(10.0f, 11.0f, 12.0f,
-                                          13.0f, 14.0f, 15.0f,
-                                          16.0f, 17.0f, 18.0f);
+            const sf::Transform      transform(10.0f, 11.0f, 12.0f, 13.0f, 14.0f, 15.0f, 16.0f, 17.0f, 18.0f);
             const std::vector<float> matrix(transform.getMatrix(), transform.getMatrix() + 16);
-            CHECK(matrix == std::vector<float>{10.0f, 13.0f, 0.0f, 16.0f,
-                                               11.0f, 14.0f, 0.0f, 17.0f,
-                                                0.0f,  0.0f, 1.0f,  0.0f,
-                                               12.0f, 15.0f, 0.0f, 18.0f});
+            CHECK(
```

Comment:
In situations like this I think the previous formatting was helpful because it allows you to see the data in its matrix layout, but after formatting it's harder to notice things like this

Reply:
Formatting matrices is maybe the canonical use case for `// clang-format off` but we'd basically have to wrap this entire file in a `// clang-format off` block if we wanted to format all matrices as blocks.